functions/Get-DbaBackupHistory.ps1

```diff
@@ -258,18 +284,17 @@ function Get-DbaBackupHistory {
 							continue
 						}
 					}
-					$Allbackups += $Logdb = Get-DbaBackupHistory -SqlInstance $server -Databases $db -raw:$raw -DeviceType $DeviceType -LastLsn $TLogstartLSN | Where-Object { 
+					$Allbackups += $Logdb = Get-DbaBackupHistory -SqlInstance $server -Database $db.Name -raw:$raw -DeviceType $DeviceType -LastLsn $TLogstartLSN | Where-Object { 
 						$_.Type -eq 'Log' -and [bigint]$_.LastLsn -gt [bigint]$TLogstartLSN -and [bigint]$_.DatabaseBackupLSN -eq [bigint]$Fulldb.CheckPointLSN 
-					} | Sort-Object LastLsn
-					$Allbackups
+					}
+					$Allbackups | Sort-Object FirstLsn
```

Comment:
Any reason for switching from LastLsn to FirstLsn? (shouldn't make a difference)